Products/ZPsycopgDA/dtml/add.dtml

```diff
@@ -1,26 +1,23 @@
 <dtml-var manage_page_header>
 
 <dtml-var "manage_form_title(this(), _,
-           form_title='Add Z Psycopg 2 Database Connection',
-           help_product='ZPsycopgDA',
```

Comment:
Why did you remove these?

Reply:
And the title stayed. So, this is actually OK, as far as I can see.

Reply:
Do you have a pointer to the definition of `manage_form_title` so I can understand why it does not do much?

Reply:
The `manage_form_title` is actually a DTMLMethod with source code at `App/dtml/manage_form_title.dtml`.

There, if `help_product` and `help_topic` are both present, a help link is produced, by using a method from `HelpSys/HelpSys.py`.

The link uses the product name and the topic to build an URL which points at the structured text in the `help` directory.

The product "ZPsycopgDA" does not include a `help` directory. Therefore, this link is useless and the settings should be removed.